lib/registrant.rb

```diff
@@ -0,0 +1,18 @@
+class Registrant
+  attr_reader :name,
+              :age,
+              :permit,
+              :license_data
+  attr_writer :license_data
+  
+  def initialize(name, age, permit = false)
+    @name = name
+    @age = age
+    @license_data = { written: false, license: false , renewed: false }
+    @permit = permit
+  end
```

Comment:
You've defined an `attr_writer` for `@license_data`, which allows direct modification of the license_data hash from outside the class. This is only necessary if you want to allow `@license_data` to point to something other than the hash it is currently pointing to. 

Because `@license_data` is pointing to a hash and hashes can be updated without the `attr_reader`, line `6` can be removed.